### PR TITLE
Sources Sync custom metrics

### PR DESCRIPTION
**Pull request by slemrmartin**
**Issue**: https://github.com/RedHatInsights/topological_inventory-api/issues/320

Added custom metrics:
* topological_inventory_sources_sync_errors_total
* topological_inventory_sources_sync_sources_created
* topological_inventory_sources_sync_sources_destroyed

### CODE REVIEW

**Review by lindgrenj6 — CHANGES_REQUESTED** [review]

File: spec/topological_inventory/sync/metrics_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+RSpec.describe TopologicalInventory::Sync::Metrics do
```
[the remark this answers is not in the document]
Comment (slemrmartin) [reply spec/topological_inventory/sync/metrics_spec.rb:1]: @lindgrenj6 this is different than openshift's test. It's testing only this class and never reaches any functionality of prometheus exporter gem

[the remark this answers is not in the document]
Comment (syncrou) [reply spec/topological_inventory/sync/metrics_spec.rb:1]: @lindgrenj6 do you have any follow up to https://github.com/RedHatInsights/topological_inventory-sync/pull/72#discussion_r503692912

[the remark this answers is not in the document]
Comment (lindgrenj6) [reply spec/topological_inventory/sync/metrics_spec.rb:1]: Nope, it looks good. I didn't read close enough to notice it's mocking it.

**Review by lindgrenj6 — APPROVED** [review]

**Review by syncrou — APPROVED** [review]
Only comment here. I'd like to add a ( or the ) Jira ticket here and add the PR to Jira.

LGTM 👍

**Pull request merged**